src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -12711,15 +12711,17 @@ public void InterpolatedStringsAddedUnderObjectAddition_DefiniteAssignment()
             );
         }
 
-        [Fact]
-        public void ParenthesizedAdditiveExpression_01()
+        [Theory]
+        [InlineData(@"($""{i1}"" + $""{i2}"") + $""{i3}""")]
```

Comment:
Consider adding more complicated/deep additive expressions to test diverse tree shapes.
Also consider testing one very deep additive expression (that benefits from non-recursive design).